web/pdf_presentation_mode.js

```diff
@@ -183,11 +183,34 @@ class PDFPresentationMode {
    * @private
    */
   _notifyStateChange() {
-    this.eventBus.dispatch("presentationmodechanged", {
-      source: this,
-      active: this.active,
-      switchInProgress: !!this.switchInProgress,
-    });
+    let state = PresentationModeState.NORMAL;
+    if (this.switchInProgress) {
+      state = PresentationModeState.CHANGING;
+    } else if (this.active) {
+      state = PresentationModeState.FULLSCREEN;
+    }
+
+    if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("MOZCENTRAL")) {
```

Comment:
Shouldn't this be `typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")` instead since the throwing is only for third-party usage and therefore not needed if we are targeting mozcentral?

Reply:
D'oh; of course it should :-)

Embarrassingly enough I managed to copy-paste the wrong thing here, thanks for spotting this!